docker-compose.yml

```diff
@@ -78,20 +78,18 @@ nginx:
 
 # Consul is the service catalog that helps discovery between the components
 consul:
-  image: progrium/consul:latest
-  command: -server -bootstrap -ui-dir /ui
+  image: autopilotpattern/consul:latest
+  command: >
+    /usr/local/bin/containerpilot
+    /bin/consul agent -server
+    -bootstrap-expect 1
```

Comment:
@tgross this works to create a one-node Consul cluster that we can then scale to N nodes via Docker Compose.